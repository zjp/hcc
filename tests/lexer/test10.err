FATAL [2,1]: Bad escape sequence in char literal
FATAL [2,9]: Bad escape sequence in char literal
FATAL [2,13]: Bad escape sequence in char literal
FATAL [2,17]: Bad escape sequence in char literal
FATAL [2,21]: Bad escape sequence in char literal
