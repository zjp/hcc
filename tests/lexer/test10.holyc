int onetwo = 12; char charchar = '\t; hahahah; '/
'\s '\t '\e '\a '\w '\k '\ '\	
