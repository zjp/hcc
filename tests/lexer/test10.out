INT [1,1]
ID:onetwo [1,5]
ASSIGN [1,12]
INTLIT:12 [1,14]
SEMICOLON [1,16]
CHAR [1,18]
ID:charchar [1,23]
ASSIGN [1,32]
CHARLIT:tab
SEMICOLON [1,37]
ID:hahahah [1,39]
SEMICOLON [1,46]
CHARLIT:/
CHARLIT:tab
CHARLIT: 
CHARLIT:tab
EOF [3,1]
